### PR TITLE
Add 0.163 release notes

### PR DESCRIPTION
**Pull request by martint**

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.163.rst [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Release 0.163
+=============
+
+General Changes
+---------------
+
+* Improve performance for ``OUTER JOIN`` queries.
+* Improve exchange performance by reading from buffers in parallel.
+* Improve performance when only a subset of the columns resulting from a ``JOIN`` are referenced.
+* Make ``ALL``, ``SOME``, and ``ANY`` non-reserved keywords.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.163.rst]: We don't normally put a comma before and

**Comment by martint** [comment]
> There are additional changes:

Yeah, I'm still waiting for people to send me their notes.

**Comment by martint** [comment]
> Avoid variable boxing in InMemoryHashAggregationBuilder (improves GC for aggregations?)

It's only for spilling.

File: presto-docs/src/main/sphinx/release/release-0.163.rst [hunk 2]
```diff
@@ -0,0 +1,43 @@
+=============
+Release 0.163
+=============
+
+General Changes
+---------------
+
+* Fix data corruption when transporting dictionary-encoded data.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.163.rst]: Maybe say "Fix data corruption that causes incorrect results when ..."

Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.163.rst]: @dain, can this produce incorrect results? or just plain failures?

**Pull request closed without merging**